qiskit_metal/renderers/renderer_gmsh/gmsh_renderer.py

```diff
@@ -368,17 +426,8 @@ def render_element_junction(self, junc: pd.Series):
         """
         qc_shapely = junc.geometry
         qc_width = self.parse_units_gmsh(junc.width)
-        props = ["thickness", "z_coord"]
-        result = self.parse_units_gmsh(
-            self.design.ls.get_properties_for_layer_datatype(
-                properties=props, layer_number=junc.layer))
-        if result:
-            qc_thickness, qc_z = result
-        else:
-            raise ValueError(
-                f"Could not find {props} for the layer_number={junc.layer} in component"
-                f" '{junc['component'].name}'. Check your design and try again."
-            )
+        qc_thickness, qc_z = self.get_thickness_zcoord_for_layer_datatype(
+            layer_num=junc.layer)
```

Comment:
Eventually, data may be in the qgeometry column.  Even, though the default is zero for now, that could change.

What do you think about adding data_type = 0 for now?

Reply:
The datatype argument is zero by default as can be seen in line 161. Hence, I didn't pass anything here. Hope that okay?

Reply:
Sounds good.